geodiff/src/geodiff.cpp

```diff
@@ -230,8 +230,21 @@ int GEODIFF_applyChangeset( const char *base, const char *changeset )
   }
 }
 
-int GEODIFF_createRebasedChangeset( const char *base, const char *modified, const char *changeset_their, const char *changeset )
+int GEODIFF_createRebasedChangeset( const char *base,
+                                    const char *modified,
+                                    const char *changeset_their,
+                                    const char *changeset,
+                                    const char *conflictfile,
+                                    int *nConflicts )
```

Comment:
I am wondering if it would be better not to return nConflicts here - and maybe instead have a function within tests to return number of conflicts if needed? But no strong opinion...

Reply:
thinking about input/mergin client. They do rebase and get conflict file. if you return nconflict, you can directly do some action. otherwise you would need to parse JSON to see if there are conflicts or not...

@varmar05 what do you think, would you find nConflict directly here usefull? Or alternatively, A) we do not need to create conflicfile at all in case there are no conflicts, or B) have some API that takes conflict file and returns you number of conflicts/if it has conflicts at all

Reply:
maybe A) to create and return conflict file only if there are some?

Reply:
done